test/loopback.test.js

```diff
@@ -631,4 +631,73 @@ describe('loopback', function() {
       });
     });
   });
+
+  describe('new remote method configuration', function() {
+    it('Remote method - isStatic specified in method name', function() {
```

Comment:
Per my previous comment, this would be a better test name IMO:

```js
it('treats method names starting with "prototype." as "isStatic:false"', ...)
```